docs/use-cases/401k.md

```diff
@@ -7,9 +7,209 @@ parent: Use Cases
 
 ## 401k
 
-Learn about some sample queries to perform common 401k actions
+Intuit Payroll is enengaged  in partnerships with 401(k) plan providers to offer 401(k) benefit to Small Business Owners. This will allow businesses to provide retirement benefits to their employees. This guide will help you understand the high level integration.
 
+### Integraton Steps
 
-### Details
+* Create and configure app on Inuit Developer portal: [pre-production](https://developer-stage.intuit.com/app/developer/homepage) (do this first) and [production](https://developer.intuit.com/app/developer/homepage)  (Note: Use existing app unless there is a strong need to switch to new app)
+  * [get started](https://developer.intuit.com/app/developer/qbo/docs/get-started), [go live](https://developer.intuit.com/app/developer/qbo/docs/go-live)  
+* Implement OpenId Connect
+  * [OpenID Connect](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/openid-connect) integration - send scopes openId+email+profile (Additional scopes can be requested if needed).
+  * Send claims in the request (claims={"id_token":{"realmId":null}}) - OpenIdConnect doesn’t return back QBO realm info, by passing the claims in the request Intuit will send back the realmid if user is logged in. 
+  * Call [UserInfo endpoint](https://developer.intuit.com/app/developer/qbo/docs/develop/authentication-and-authorization/openid-connect#obtaining-user-profile-information) to obtain user profile information.
+  * Call [Role API]() - UserInfo and RoleAPI can be called in parallel
+* Make OAuth call by sending accounting+payroll scope
+  * Send realm_id in the request (&realm_id=<realm id>) if available, this will suppress the company picker screen.
+
+* Call Payroll APIs
+  * Payroll GraphGL endpoints
+
+
+### Use Cases
+
+There are four major connection points into payroll for the third party 401(k) providers 
+
+#### Company Setup
+
+When first setting up a company with a new 401k provider, information about the company needs to be sent back to map the authorization to the information in their system. Here are the fields as they have described it.
+
+* Company ID
+* EIN
+* Company Name
+* Company Address (primary address -> formatted address)
+* Admin Name (business owner or signing authority if needed)
+* Admin Email
+
+This information is particularly helpful because if the company owner decides to subscribe to a 401k provider through the QB App Store, then this information will be pre-filled for them. And if the 401k provider has been authorized they can pull an employee roster which will help their recommendation engine about what kind of plan to pick for everyone. 
+
+After eestablishing connecton with QBO company run this  query for company details
+
+```GRAPHQL
```

Comment:
Ran a few tests on local with the `GRAPHQL` type on the code block, to see what it looks like.

GRAPHQL (all caps) looks a bit strange:
<img width="608" alt="Screen Shot 2020-12-23 at 11 19 47 AM" src="https://user-images.githubusercontent.com/1093919/103030666-1e4c2780-4511-11eb-8682-088b6d16d770.png">

graphql (lowercase) is a bit better, but greyed out:
<img width="565" alt="Screen Shot 2020-12-23 at 11 20 00 AM" src="https://user-images.githubusercontent.com/1093919/103030667-1e4c2780-4511-11eb-908c-13ac691a9d8f.png">

And for reference, this is no qualifier on the code block:
<img width="575" alt="Screen Shot 2020-12-23 at 11 20 15 AM" src="https://user-images.githubusercontent.com/1093919/103030669-1e4c2780-4511-11eb-85a1-8e760b235a8e.png">

My vote would probably be either lowercase or no qualifier, but open to whatever you think is best here.